pagerduty/service_event_orchestration_active.go

```diff
@@ -0,0 +1,65 @@
+package pagerduty
+
+import "fmt"
+
+type ServiceEventOrchestrationActiveService service
+
+type ServiceEventOrchestrationActive struct {
+	Active bool `json:"active"`
+}
+
+// Create switches service orchestration to desired status.
+func (s *ServiceEventOrchestrationActiveService) Create(serviceID string, sActive *ServiceEventOrchestrationActive) (*ServiceEventOrchestrationActive, *Response, error) {
+	u := fmt.Sprintf("/event_orchestrations/services/%s/active", serviceID)
+	v := new(ServiceEventOrchestrationActive)
+	p := ServiceEventOrchestrationActive{Active: sActive.Active}
+
+	resp, err := s.client.newRequestDo("PUT", u, nil, p, v)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return v, resp, nil
+}
+
+// Get gets the current status of service orchestration.
+func (s *ServiceEventOrchestrationActiveService) Get(serviceID string) (*ServiceEventOrchestrationActive, *Response, error) {
+	u := fmt.Sprintf("/event_orchestrations/services/%s/active", serviceID)
+	v := new(ServiceEventOrchestrationActive)
+	p := &ServiceEventOrchestrationActive{}
+
+	resp, err := s.client.newRequestDo("GET", u, nil, p, v)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	return v, resp, nil
+}
+
+// Delete switches service orchestration to inactive status.
+func (s *ServiceEventOrchestrationActiveService) Delete(serviceID string) (*Response, error) {
+	u := fmt.Sprintf("/event_orchestrations/services/%s/active", serviceID)
+	v := new(ServiceEventOrchestrationActive)
+	p := &ServiceEventOrchestrationActive{Active: false}
+
+	resp, err := s.client.newRequestDo("PUT", u, nil, p, v)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+// Update updates service orchestration to desired status.
+func (s *ServiceEventOrchestrationActiveService) Update(serviceID string, sActive *ServiceEventOrchestrationActive) (*ServiceEventOrchestrationActive, *Response, error) {
```

Comment:
Do we actually need three separate methods to change the event orchestration status? Maybe we could just have `Set` method that would set the desired status instead of `Create`, `Delete` and `Update`?

If the library requires having these 3 methods, I'd maybe create a separate helper method that could be used by `Create`, `Delete` and `Update`